tests/foreman/api/test_errata.py

```diff
@@ -1406,22 +1407,68 @@ def test_errata_installation_with_swidtags(
     """
     module_name = 'kangaroo'
     version = '20180704111719'
-    # setup base_rhel8 and sat_tools repos
-    rhel8_contenthost.create_custom_repos(
-        **{
-            'baseos': settings.repos.rhel8_os.baseos,
-            'appstream': settings.repos.rhel8_os.appstream,
-        }
+    # new cv version for ak
+    cv_publish_promote(
+        target_sat,
+        module_sca_manifest_org,
+        module_cv,
+        module_lce,
     )
-    function_repos_collection_with_manifest.setup_virtual_machine(rhel8_contenthost)
-
-    # install older module stream
+    _ak = target_sat.api.ActivationKey(
+        organization=module_sca_manifest_org,
+        environment=module_lce,
+        content_view=module_cv,
+    ).create()
+    # needed repos for module stream, swid tags, prereqs
+    _repos = {
+        'base_os': settings.repos.rhel8_os.baseos,
+        'sat_tools': settings.repos.rhel8_os.appstream,
+        'swid_tags': settings.repos.swid_tag.url,
+    }
+    # associate repos with host, org, lce, and sync
+    rhel8_contenthost.create_custom_repos(**_repos)
+    for _key in _repos:
+        target_sat.cli_factory.setup_org_for_a_custom_repo(
+            {
+                'url': _repos[_key],
+                'name': _key,
+                'organization-id': module_sca_manifest_org.id,
+                'lifecycle-environment-id': module_lce.id,
+                'activationkey-id': _ak.id,
+            },
+        )
+    # promote new version with all repos/content
+    cv_publish_promote(
+        target_sat,
+        module_sca_manifest_org,
+        module_cv,
+        module_lce,
+    )
+    # register host with ak, succeeds
+    result = rhel8_contenthost.register(
+        activation_keys=_ak.name,
+        target=target_sat,
+        org=module_sca_manifest_org,
+        loc=None,
+    )
+    assert result.status == 0, f'Failed to register the host {target_sat.hostname},\n{result}'
     rhel8_contenthost.add_rex_key(satellite=target_sat)
+    assert (
+        rhel8_contenthost.subscribed
+    ), f'Failed to subscribe the host {target_sat.hostname}, to content.'
+    rhel8_contenthost.install_katello_ca(target_sat)
```

Comment:
This should not be needed after global registration I believe.
```suggestion
```

Reply:
you're right seems it isn't needed, removing for next push!